composer.json

```diff
@@ -112,7 +112,7 @@
         "psr-4": {
             "App\\": "app/",
             "Database\\Factories\\": "database/factories/",
-            "Database\\Seeders\\": "database/seeders/"
+            "Database\\Seeders\\": "database/seeder/"
```

Comment:
change the folder name to seeders instead. keep it as close to laravel file structure

Reply:
view pyrocms @diashalabi 

https://github.com/pyrocms/pyrocms/blob/208b973407e6d0728055f0c1bc4b7f6ff577a640/composer.json#L110